### ISSUE TITLE
New feature request: schedule nutritions like workout schedule

### ISSUE THREAD

**Issue by ATSGIT** [wger-project/wger#358]
Hi,
I'm not sure what is the purpose of "Nutrition plan" in the app. Could you please tell me more about it. I didn't find anything in the documentation!
Anyway, I think ,as a bodybuilder, it should be a feature where trainer be able to define and **schedule** "nutrition plans" like what s/he do in the "workout schedule". In this way, trainee will be able to choose and eat foods based on what his/her trainer specified for

For example, trainer schedule the following plan for trainee:
Sunday: 3 eggs in 7:00, 30g Protein supplement in 11:00 am. etc
Monday: 100g Cereals, 50g Cheese in 7:00
etc.

**Comment by rolandgeider** [comment]
You are right that, at the moment the nutrition plans are not so flexible, so have a plan and that's it: you follow it till you get another one. I can also thing of other things that could be useful, such as differentiating between training and non-training days. Would you be interested in working on this?
